content/applications/inventory_and_mrp/manufacturing/plm/approvals.rst

```diff
@@ -0,0 +1,151 @@
+:guilabel:`Approvals`. Type in the approver's position or title under :guilabel:`Role` (e.g.
+"Engineering Manager", "Quality Team"), and select the relevant :guilabel:`User` from the drop-down
+menu. Set the :guilabel:`Approval Type` to :guilabel:`Is required to approve`, :guilabel:`Approves,
+but the approval is optional`, or :guilabel:`Comments only`.
+
+.. example::
+   Assign the `CTO`, "Mitchell Admin" as a required approver for |ECOs| in the `Validated` stage in
+   the `New Product Introduction` ECO type. Approvals from the quality and marketing teams are not
+   required to apply changes to the |ECO| because their :guilabel:`Approval Type` is set to
+   :guilabel:`Approves, but the approval is optional` and :guilabel:`Comments only`, respectively.
+
+   .. image:: approvals/approvers.png
+      :align: center
+      :alt: Set an approver that "Is required to approve" ECOs in the "Validated" stage.
+
+Manage approvals
+================
+
+Approvers can easily track their to-do approvals by navigating to the :menuselection:`PLM app` and
+looking at the ECO type card, which shows the count of open tasks assigned to them. Here's what each
+button on the ECO type card does:
+
+#. :guilabel:`X Engineering Changes` button displays a count of in-progress |ECOs| of this ECO type.
+   Clicking the button opens the Gantt view of the :guilabel:`Engineering Change Orders` page.
+
+#. :guilabel:`My Validations` displays a count of |ECOs| that the approver must accept or reject.
+   Clicking on this button displays |ECOs| pending approval or rejected (marked with the red
+   :guilabel:`Blocked` blocked state).
+
+#. :guilabel:`All Validations` displays a count of |ECOs| a verification stage pending approval from
+   any approvers. Clicking on the button displays |ECOs| pending approval from or rejected by any
+   approver.
+
+#. :guilabel:`To Apply` displays a count of |ECOs| the user needs to apply changes to. Clicking on
+   the button displays all the |ECOs| to approve and apply changes to in the verification stage.
+   |ECOs| marked with the green :guilabel:`Done` stage have already been approved, and the user just
+   needs to click on the |ECO| to enter the form view and click the :guilabel:`Apply Changes`
+   button.
+
+.. image:: approvals/validation-overview.png
+   :align: center
+   :alt: Display count of validations to-do and buttons to open filtered list of ECOs.
+
+Approve ECOs
+------------
+
+Navigate to an |ECO| in a verification stage while logged in as the assigned approval to see the
+:guilabel:`Approve`, :guilabel:`Reject`, and :guilabel:`Apply Changes` button. To approve the |ECO|
+and apply the changes onto the production :abbr:`BoM (Bill of Materials)`, click :guilabel:`Approve`
+then :guilabel:`Apply Changes`. Note that the :guilabel:`Apply Changes` button will not work unless
+the :guilabel:`Approve` button was pressed first. Additionally, the chatter will log the history of
+the buttons pressed.
```

Comment:
apply changets button will not work until all approvers have approvede. (When there are multiple approvers)